M2/Macaulay2/packages/Complexes/ChainComplexMap.m2

```diff
@@ -99,13 +99,9 @@ map(Complex, Complex, ZZ) := ComplexMap => opts -> (D, C, j) -> (
         result.cache.isCommutative = true;
         return result
         );
-    if j === 1 then (
-        if C == D and (opts.Degree === null or opts.Degree === 0) then
-            return id_C;
-        error "expected source and target to be the same";
-        );
-    error "expected integer to be zero or one";
-    )
+    if C == D and (opts.Degree === null or opts.Degree === 0) then
+        return j * id_C;
+    error "expected 0 or source and target to be the same")
```

Comment:
I think that this change is already part of a pull request associated to ``Complexes``.

Reply:
Yes, it's the same commit, so once one of them is merged, the duplicate commit will be skipped in the other PR.